extensions/amp-access/0.1/login-dialog.js

```diff
@@ -267,7 +267,7 @@ export class WebLoginDialog {
     }
     dev().fine(TAG, 'Login done: ', result, opt_error);
     if (opt_error) {
-      this.reject_(opt_error);
+      this.reject_(user().createError('login failed', opt_error));
```

Comment:
Seems a bit brittle w.r.t. future changes. I'm not sure how valuable "login failed" since we do have error stack. If it is, maybe something like the current `createErrorVargs` in `log.js` better? It extends the error message but preserves user/dev flag.